src/main/scala/com/databricks/labs/overwatch/Optimizer.scala

```diff
@@ -50,27 +50,40 @@ object Optimizer extends SparkSessionWrapper{
 
   /**
    * pass in the overwatch ETL database name to optimize overwatch in parallel
-   * @param args ["overwatch_etl"]
+   * @param args(0)   Overwatch ETL Database.
+   * @param args(1)   Organization_ID for which optmization need to be performed.(Optional). By default optimizer will run for all
+   *                  the organization ID in pipeline_report.
    */
   def main(args: Array[String]): Unit = {
 
-    val overwatchETLDB = if (args.length == 1) {
-      args(0)
+
+    val (overwatchETLDB, orgIDs) = if (args.length == 1) {
+      val orgIdList = spark.table(s"${args(0)}.pipeline_report").select("organization_id").distinct().collect().map(x => x(0).toString)
```

Comment:
for readability, assign args(0) to a variable and then use that

Reply:
@gueniai this was already there in previous PR commit. But as per @sriram251-code I have changed this in this way. Do you want me to change it to val orgId = args.lift(1).getOrElse(" ") in this way.